asset_dashboard/models.py

```diff
@@ -64,33 +64,42 @@ def __str__(self):
 class ProjectScore(models.Model):
 
     project = models.OneToOneField(Project, on_delete=models.CASCADE)
-    core_mission_score = models.IntegerField(default=1,
+    core_mission_score = models.IntegerField(null=True, blank=True,
                                              validators=[MinValueValidator(1),
                                                          MaxValueValidator(5)])
-    operations_impact_score = models.IntegerField(default=1,
+    operations_impact_score = models.IntegerField(null=True, blank=True,
                                                   validators=[MinValueValidator(1),
                                                               MaxValueValidator(5)])
-    sustainability_score = models.IntegerField(default=1,
+    sustainability_score = models.IntegerField(null=True, blank=True,
                                                validators=[MinValueValidator(1),
                                                            MaxValueValidator(5)])
-    ease_score = models.IntegerField(default=1,
+    ease_score = models.IntegerField(null=True, blank=True,
                                      validators=[MinValueValidator(1),
                                                  MaxValueValidator(5)])
-    geographic_distance_score = models.IntegerField(default=1,
+    geographic_distance_score = models.IntegerField(null=True, blank=True,
                                                     validators=[MinValueValidator(1),
                                                                 MaxValueValidator(5)])
-    social_equity_score = models.IntegerField(default=1,
+    social_equity_score = models.IntegerField(null=True, blank=True,
                                               validators=[MinValueValidator(1),
                                                           MaxValueValidator(5)])
 
     @property
     def total_score(self):
-        '''return the total, weighted score'''
-        ...
+        # get the ProjectScore fields, excluding the id and project fields
+        score_fields = [f for f in self._meta.get_fields() if f.name != 'id' if f.name != 'project']
+        score_weights = ScoreWeights.objects.all()[0]
```

Comment:
does it work to do `ScoreWeights.objects.get()`

if so, that would be preferred because it will be raises a useful error if there is more than one score weight

Reply:
that's what i originally had, then my tests failed because that pk didn't exist. is there a way to get the primary key, dynamically?

Reply:
not off the top of my head. let's do it this way

```python
# there should be one, and only one row in score weights. 
# this tuple unpacking will throw an error if that's not so
score_weights, = ScoreWights.objects.all()
````

Reply:
cool, today i learned about the tuple unpack error. saving this snippet. commit here: 970e47e

Reply:
improved the test here: 5d3d220